pkg/asset/agent/image/oc.go

```diff
@@ -76,16 +76,21 @@ func (r *release) GetBaseIso(log logrus.FieldLogger, releaseImage, pullSecret, a
 	if err != nil {
 		return "", err
 	}
-
-	filename := fmt.Sprintf(coreOsFileName, architecture)
+	switch rhcosArch {
+	case "arm64":
+		rhcosArch = "aarch64"
+	case "amd64":
+		rhcosArch = "x86_64"
+	}
```

Comment:
This code exists in multiple places now, so we should define a function to get the architecture name used by CoreOS given the name in the install-config.